README.md

```diff
@@ -52,6 +52,18 @@ There are 4 key features we plan to implement in the near term.
 
 For more detail on these features and where we are going, see [our roadmap](ROADMAP.md).
 
+## supported and experimental usage modes
```

Comment:
This title does not relate to the content. I think the title should directly refer to hosting cellxgene.

Something like "[Unsupported] Hosting cellxgene"